finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,143 @@
+function frogs(numInitialFrogs) {
+  var rhyme = '';
```

Comment:
I might add a guard clause here to say if the number is a thousand or more, return a warning that it's too large of a number - since your word conversion isn't setup for that